inference_worker.py

```diff
@@ -416,12 +421,21 @@ def _worker_thread(self, thread_id):
             pass
 
         dbg("running worker", thread_id)
+        # Don't start counting till we've warmed up.
+        batches = -500
```

Comment:
Can you remove the special case logic for warming up please? If we're going to start using the inference worker to play via GTP, we really need to perform a warm up inference during initialization, similar to [tf_dual_net.cc](https://github.com/tensorflow/minigo/blob/a3e1df9cc9802b224db6092257245e38e10aa746/cc/dual_net/tf_dual_net.cc#L71), otherwise the time estimation will be all wrong.

Reply:
~I've noticed it takes a couple batches for the whole system to hit steady state, this just delays the initial "start" time by a couple hundred batches so that long term throughput is a little more stable calculation. I can certainly reduce this to -100 (or even -10).~

KISS, I'll remove this.